index.js

```diff
@@ -65,7 +53,7 @@ module.exports = function (a, b) {
 	var j = 0;
 
 	while (i < aLen) {
-		charCodeCache[start + i] = a.charCodeAt(start + i);
```

Comment:
It is not necessary to add `start` here to the `charCodeCache`, as long as we do the same in the loop.

This make the bench ~4-5% faster on my cpu